### ISSUE TITLE
Support asciinema casts

### ISSUE THREAD

**Issue by waldyrious** [isaacs/github#1221]
In addition to supporting rich text rendering of various markup formats, listed [here](https://github.com/github/markup/blob/master/README.md#markups),
GitHub currently also supports rich rendering of a bunch of binary files, such as images, STL files, PDF files, etc., as described [here](https://help.github.com/categories/working-with-non-code-files/).

A recurring (and IMO growing) pattern on GitHub is projects using animated GIFs in README files and other documentation to illustrate installation or usage workflows.
When these animations are of code editing, command-line interaction, or similar tasks, a nicer alternative would be to use [asciinema](https://asciinema.org/) casts.
However, these are not rendered inline, which makes their use quite cumbersome.

Please add support for embedding of live asciinema casts in rich-text renderings of document files (markdown, etc.).

**Comment by Mottie** [comment]
Did you contact GitHub about this request? You may also want to open an issue in https://github.com/github/markup/ to start a discussion.

**Comment by waldyrious** [comment]
> Did you contact GitHub about this request?

No, I haven't. I just wanted to put this out so that there'd be a public place for tracking this feature, which a direct contact wouldn't provide.

As for github/markup, I did look at that repo, but it seems to be related to the rich text rendering of lightweight markup languages, rather than to the rich embedding of non-textual content.
